libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/placeslistonmap/PlacesListItemClickListener.kt

```diff
@@ -1,7 +1,9 @@
 package com.mapbox.androidauto.car.placeslistonmap
 
+import androidx.annotation.UiThread
 import com.mapbox.androidauto.car.search.PlaceRecord
 
+@UiThread
```

Comment:
Since it is an `interface` that doesn't have any implementation, why does it have this requirement?

Reply:
Our implementation of this interface calls the `@UiThread` annotated method. See `PlacesListOnMapScreen#placeClickListener`.